causely/manifest.json

```
@@ -23,17 +23,20 @@
       },
       {
         "media_type": "image",
-        "caption": "Causelhy Remediated Root Causes",
+        "caption": "Causely Remediated Root Causes",
         "image_url": "images/Causely_Remediated_RootCauses.png"
       }
     ],
     "classifier_tags": [
-      "Category::Containers",
-      "Category::Kubernetes",
+      "Category::AI",
```

```suggestion
      "Category::AI/ML",
```
fixing incorrect category name

---

thank you! fixed